core/src/main/java/org/testcontainers/containers/DockerComposeContainer.java

```diff
@@ -530,6 +531,17 @@ public void invoke() {
         }
     }
 
+    // #460 : docker-compose are not resolvable on Win10 machines because there is docker-compose.exe binary,
```

Comment:
I think we can omit this comment (or change it to correct Javadoc comment syntax, but I prefer omitting it).

Reply:
Agree